bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolderRenderer.java

```diff
@@ -1696,9 +1696,8 @@ Color getFillColor() {
 
 	private Font getChevronFont(Display display) {
 		if (chevronFont == null) {
-			Point dpi = display.getDPI();
-			int fontHeight = 72 * CHEVRON_FONT_HEIGHT / dpi.y;
 			FontData fd = parent.getFont().getFontData()[0];
+			int fontHeight = (int) (fd.height * 0.7);
```

Comment:
This class has lots of magic numbers. The "0.7" is one of them. Can we create a constant with a telling name of it?